packages/msprime/test_msprime.py

```diff
@@ -0,0 +1,26 @@
+from pyodide_test_runner import run_in_pyodide
+
+
+@run_in_pyodide(
+    packages=[
+        "numpy",
+        "svgwrite",
+        "attrs",
+        "pyrsistent",
+        "jsonschema",
+        "tskit",
+        "demes",
+        "msprime",
+        "newick",
```

Comment:
Normally you don't need to specify packages that are already dependencies of msprime. It's probably a good idea to omit those to check that dependencies are correctly installed.